nncf/torch/pruning/base_algo.py

```diff
@@ -138,7 +138,7 @@ def _prune_weights(self, target_model: NNCFNetwork):
                                  PTNNCFPruningTensorProcessor).mask_propagation()
 
         # Adding binary masks also for Batch/Group Norms to allow applying masks after propagation
-        types_to_apply_mask = ['group_norm']
+        types_to_apply_mask = ['group_norm', 'instance_norm']
```

Comment:
Please check masks are correctly applying to instance_norm weights during training

Reply:
Resolved